docs/source/nlp/question_answering.rst

```diff
@@ -0,0 +1,339 @@
+| optim.sched.name                          | string          | SquareRootAnnealing                                                              | The warmup schedule                                                                                          |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| optim.sched.warmup_ratio                  | float           | 0.0                                                                              | The warmup ratio                                                                                             |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+
+Example of the command for training the model:
+
+.. code::
+
+    python question_answering_squad.py \
+           model.train_ds.file=<PATH_TO_TRAIN_DATA_FILE>  \
+           model.validation_ds.file=<PATH_TO_VALIDATION_DATA_FILE>  \
+           model.dataset.version_2_with_negative=<ALLOW_UNANSWERABLE_SAMPLES>  \
+           model.dataset.do_lower_case=<DO_LOWER_CASE>  \
+           trainer.max_epochs=<NUM_EPOCHS> \
+           trainer.gpus=[<CHANGE_TO_GPU(s)_YOU_WANT_TO_USE>]
+
+.. Note:: The first time you are performing training, it will take an extra 5-10 minutes to process
+   the dataset for training. For future training runs, it will use the processed dataset if :code:`model.dataset.use_cache=true`, which is
+   automatically cached in the files in the same directory as the data.
+
+Required Arguments for Training
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* :code:`model.train_ds.file`: Path to the training file in JSON format.
+* :code:`model.validation_ds.file`: Path to the validation file in JSON format.
+
+Optional Arguments
+^^^^^^^^^^^^^^^^^^
+
+* Other arguments to override fields in the specification file.
+
+.. note::
+
+    While the arguments are defined in the spec file, if you wish to override these parameter definitions in the spec file \
+    and experiment with them, you may do so over command line by simple defining the param. \
+    For example, the sample spec file mentioned above has :code:`validation_ds.batch_size` set to 24. \
+    However, if you see that the GPU utilization can be optimized further by using larger a batch size, \
+    you may override to the desired value, by adding the field :code:`validation_ds.batch_size=128` over command line.
+    You may repeat this with any of the parameters defined in the sample spec file.
+
+Important parameters
+^^^^^^^^^^^^^^^^^^^^
```

Comment:
I have the same passage for other models, maybe move this to a separate NLPModel .rst along with other common params? Could you please add it?

Reply:
ok